internal/acceptance/external_location_test.go

```diff
@@ -60,3 +60,112 @@ func TestUcAccExternalLocationForceDestroy(t *testing.T) {
 		}`,
 	})
 }
+
+func TestUcAccExternalLocationUpdate(t *testing.T) {
```

Comment:
Why are we testing storage credentials changes here as well?

Reply:
We used to have these tests combined together before as well (eg: above test: `TestUcAccExternalLocationForceDestroy`), since external location needs storage credential so it removed having another tests that does that same thing but we remove code duplication